contrib/babelfishpg_tds/src/backend/tds/tdsprotocol.c

```diff
@@ -162,6 +163,14 @@ ResetTDSConnection(void)
 	TdsSendEnvChange(TDS_ENVID_RESETCON, NULL, NULL);
 }
 
+/*
+ * SetResetTDSConnectionFlag - Sets reset tds connection flag
+ */
+void SetResetTDSConnectionFlag()
```

Comment:
On diving deep, Found 2 behaviour differences:
1. sys.sp_reset_connection should not send Env change token and TDSResetConnection should send
2. Isolation levels are reset for sp_reset_connection but retained for TDSResetConnection

We have not fixed this for sp_reset_connection and this is paramount for feature complete. Lets focus on testing and aligning TSQL Behaviour and using wireshark